timesketch/__init__.py

```diff
@@ -24,7 +24,4 @@ def create_app(config=None):
 
     Returns:
         Application object (instance of flask.Flask).
-    """
-    # pylint: disable=import-outside-toplevel
-    from timesketch import app
-    return app.create_app(config)
+    """
```

Comment:
why did you delete these lines?

Reply:
hm guess they have been deleted by the refactoring, I dit not intentionally mess this this part of the init file at all.

Reply:
but you are deleting this with this PR, so I would like to ask you to revert that, otherwise TS will stop working for few tools